workflows/envs/cytominer_env.yaml

```diff
@@ -7,5 +7,6 @@ dependencies:
   - python=3.9
   - pyyaml
   - pip
+  - memray
   - pip:
-      - git+https://github.com/cytomining/pycytominer.git@1e984c6b266012e3e5a0f0620bc866c3958f0260
+      - git+https://github.com/cytomining/pycytominer.git@afe5df3db07f1c23237e90fcb5179bc0afa30032
```

Comment:
Would it make sense to remove the hash and rely on the most up to date version of Pycytominer from source?

Reply:
now that version 1.0 is released! Yes! There was a small bug before the 1.0 release where NumPy switched data types, hence the hash!